Hi.

This patch makes reinitialising mces dependant upon support for mces
actually being compiled in.

Regards,

Nigel

 suspend2.h |    2 ++
 1 files changed, 2 insertions(+)
diff -ruNp 9010-mce-config.patch-old/include/asm-i386/suspend2.h 9010-mce-config.patch-new/include/asm-i386/suspend2.h
--- 9010-mce-config.patch-old/include/asm-i386/suspend2.h	2005-12-27 20:44:01.000000000 +1000
+++ 9010-mce-config.patch-new/include/asm-i386/suspend2.h	2005-12-26 20:56:11.000000000 +1000
@@ -258,7 +258,9 @@ static inline void suspend2_arch_restore
 	do_fpu_end();
 
 	mtrr_ap_init();
+#ifdef CONFIG_X86_MCE
 	mcheck_init(&boot_cpu_data);
+#endif
 }
 	
 static inline void suspend2_arch_flush_caches(void)



_______________________________________________
Suspend2-devel mailing list
Suspend2-devel@lists.suspend2.net
http://lists.suspend2.net/mailman/listinfo/suspend2-devel

